openedx/features/course_experience/README.rst

```diff
@@ -0,0 +1,54 @@
+Course Experience
+-----------------
+
+This directory contains a Django application that provides the overall
+experience for navigating around an Open edX course.
+
+The course experience consists of a number of views:
```

Comment:
Thoughts:

- This is far better than nothing.  This could land as-is, but I also want to note my thoughts in case you want to adjust at all.
- Do you think this doc should be exhaustive, or just give a feel for the types of things that can be found here?
- What is the balance we want between heavy descriptions here vs more documentation (where needed) in the appropriate doc strings, and this page serving as more of an index/toc.
- I feel like I'd rather have this organized closer to the UI (even though that could change), and less like a flat list.

Sample organization:

**Course Home (page)**
- Welcome Message (WelcomeMessageFragmentView)
- Course Outline (CourseOutlineFragmentView)
- Course Dates (CourseDatesFragmentView)
- Course Sock (CourseSockFragmentView)
- Other sections like Course Handouts and Course Tools list (e.g reviews, updates, bookmarks, etc.)
    - Note: Should we have minimal docs here and more docs in the doc strings of each part to clarify?

**Course Tools**:
- Reviews (page)
- Updates (page)